examples/python/human_pose_tracking/requirements.txt

```
@@ -1,6 +1,5 @@
 # no 3.12 version yet (https://pypi.org/project/mediapipe/)
-# 0.10.10 no longer supports the legacy Pose model: https://github.com/rerun-io/rerun/issues/5859
-mediapipe==0.10.9 ; python_version <= '3.11'
+mediapipe>=0.10.0 ; python_version <= '3.11'
```

Did you test it with 0.10.0 as well, or should this maybe be 

```suggestion
mediapipe>=0.10.11 ; python_version <= '3.11'
```

---

I'd bump it to 0.10.11 yeah. Also, please make the same change in all media pipe examples (top of my head: gesture_recognition and face_detection, but you may have to run a global search).

---

I tested with `0.10.0`, it's the first version that works. Should we still bump it or try to find the first working version?

---

I dont have a strong feeling about, but I know is that 0.10.11 worked for the other example (it was set to that before I pined them to 0.10.9 because of this example). I'd say for an example, if a given recent version is known to work, and is compatible with our range of python version (3.8 -> 3.12), it's not a bad thing to pin. (The situation would be different for a lib, ofc.)

---

Okay, I have pinned all of them to 0.10.11; and they all work fine for me.